interceptor/transportcc.go

```diff
@@ -0,0 +1,156 @@
+package interceptor
+
+import (
+	"context"
+	"log"
+	"net/url"
+	"sync"
+	"time"
+
+	"github.com/pion/rtcp"
+	"github.com/pion/rtp"
+	"github.com/pion/sdp/v3"
+	"github.com/pion/webrtc/v3"
+)
+
+type TransportCC struct {
+	peerConnection         *webrtc.PeerConnection
+	outgoinSequenceNumbers map[uint32]uint16
+	incomingStats          interface{}
+	close                  chan struct{}
+	incomingExtensionIDs   map[uint32]uint8
+	outgoingExtensionIDs   map[uint32]uint8
+	feedbackWriter         FeedbackWriter
+}
+
+func ConfigureTransportCC(se *webrtc.SettingEngine, me *webrtc.MediaEngine) {
```

Comment:
I like this!

Maybe we can tie this in with `Bind` and the interceptor? On the `PeerConnection` creation it gets a handle and it can set things.